### PR TITLE
Add class PromMetrics to stream Historical data from a Prometheus host

### PR DESCRIPTION
**Pull request by 4n4nd**
Rename old class PromMetrics to PromMetricsLive.
Change env variable names to "FLT_...".
Change default timeframe values

### CODE REVIEW

File: metrics/prommetricslive.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import json
```
[the remark this answers is not in the document]
Comment (durandom) [reply metrics/prommetricslive.py:1]: well, stickler is right in that case. `json` is not used here

File: metrics/prommetricslive.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        prom_url = os.getenv("PROM_URL")
+        if not (prom_token or prom_url):
+            sys.exit("Error: Prometheus credentials not found")
+
+
+        prom = Prometheus(url=prom_url, token=prom_token, data_chunk='5m',stored_data='5m')
+
+        metrics_list = prom.all_metrics() # Get a list of all the metrics available from Prometheus
+
+        print("Polling Prometheus for new metric data")
```
Comment (durandom) [inline metrics/prommetricslive.py:52]: we need some sort of real logging.

File: prometheus/prometheus.py [hunk 3]
```diff
@@ -160,3 +163,45 @@ def get_current_metric_value(self, metric_name, label_config = None):
...
+            if response.status_code == 200:
+                data += response.json()['data']['result']
+            else:
+                raise Exception("HTTP Status Code {} {} ({})".format(
+                    response.status_code,
+                    requests.status_codes._codes[response.status_code][0],
+                    response.content
+                ))
+            start += chunk_seconds
+        return ((data))
```
Comment (durandom) [inline prometheus/prometheus.py:207]: why do you return a `(())` ? is that a list of lists?

File: app.py [hunk 4]
```diff
@@ -55,6 +70,7 @@ def add_scores(value):
 
     # weirdness_score.subscribe(lambda value: count = count + 1)
     weirdness_score.subscribe(add_scores)
+    weirdness_score.subscribe(print)
```
Comment (durandom) [inline app.py:73]: lets not add new `print` subscribers. this makes it only more noisy. please remove this

**Review by durandom — CHANGES_REQUESTED** [review]
some minor stuff. but other than that great.
also, would you mind squashing all those commits into one?

**Comment by 4n4nd** [comment]
Let me just close this PR and create a new one with the fixes

**Pull request closed without merging**